cypress/integration/navigation.spec.js

```diff
@@ -0,0 +1,16 @@
+describe("Routing", () => {
+  it("Navigates to other pages and clicks on New Humanoids button", () => {
+    cy.visit("http://localhost:3004");
+    cy.get("#timesheetsLink").contains("Timesheets").should("exist");
+
+    cy.get("#teamMembersLink").contains("Team members").click();
+    cy.url().should("include", "/team-members");
+    cy.get("#newHumanoidButton").contains("New Humanoid").click();
+
+    cy.get("#firstName").type("Nathalie");
+    cy.get("#lastName").type("Bruinhof");
+    cy.get("#addHumanoid").contains("Add Humanoid").click();
+
+    cy.get("#closeButton").click();
```

Comment:
Don't use the `id` directly. See the best practices for [Cypress](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements)